src/test/java/hw5/steps/AssertionSteps.java

```diff
@@ -0,0 +1,110 @@
+package hw5.steps;
+
+import hw5.context.Context;
+import hw5.pages.DifferentElementsPage;
+import hw5.pages.UserTable;
+import io.cucumber.datatable.DataTable;
+import io.cucumber.java.en.Then;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
+public class AssertionSteps {
+    private DifferentElementsPage differentElementsPage =
+            new DifferentElementsPage(Context.getInstance().getDriver());
+    private UserTable userTable =
+            new UserTable(Context.getInstance().getDriver());
+
+    @Then("Name should be displayed and equals to expected username {string}")
+    public void name_should_be_displayed_and_equals_to_expected_username(String string) {
```

Comment:
Please provide more clear name for the method parameter

Reply:
Done